src/main/java/edu/cornell/kfs/module/ar/batch/service/CuVerifyBillingFrequencyService.java

```diff
@@ -8,8 +8,10 @@
 import org.kuali.kfs.module.ar.batch.service.VerifyBillingFrequencyService;
 import org.kuali.kfs.module.ar.businessobject.BillingPeriod;
 
+//CUMod: KFSPTS-15655 required this interface exist with no content.
```

Comment:
since the interface has content, should this comment be updated, if we want/need to keep it.

Reply:
Should the mod that gave this interface content was ever removed or needed to be provided to a partner school, then the interface would still need to exist with no content for that original change. That was the intent of the past tense in this comment.

Reply:
OK, I understand now ..... I am not sure how to make that clearer, so lets keep it as is